### DISCUSSION TITLE
Q: Using exclusion mask feature vs excluding regions in downstream analyses

### DISCUSSION THREAD

**Discussion by isinaltinkaya** [popsim-consortium/stdpopsim#1290] [General]
Hi,

I became aware of the `exclusion_mask` while reading through your github issues. It was recommended for excluding the low recombination regions when we use realistic recombination maps like HapMap. I want to use the exclusion mask functionality but it is currently not a part of a stable release.

I have a few questions:
* Is there a difference in theory between using exclusion mask or excluding these regions in downstream analyses?
* At the moment, would you recommend using exclusion mask or masking it during downstream analyses?

Thanks!
Best,
Isin

**Comment by petrelharp** [comment]
Well, let's see - we're planning to make a release sometime this summer. I see from your other issue you're trying out the github version; that's fine as long as you're OK with things possibly changing. (We have no plans to change the masking stuff, though.)

> Is there a difference in theory between using exclusion mask or excluding these regions in downstream analyses?

Well, that depends what you do, I think? The masked tree sequence has just "missing data" in the masked regoins; so as long as your analyses do the right thing with that then you're OK. For instance, if you write out to VCF then that will be equivalent to writing out the un-masked VCF and removing rows in the masked region. To compute statistics using the tree sequence itself correctly you need to do some extra stuff (since at present these don't deal with missing data appropriately), and I'm happy to help with that (maybe over as a tskit discussion?).

> At the moment, would you recommend using exclusion mask or masking it during downstream analyses?

Well, the second thing is more stable and certain, so maybe masking it downstream?

**Reply by petrelharp** [reply]
I've converted this to a discussion; if this addresses your question, could you mark it as an answer? Thanks!
